src/css/_typography.scss

```diff
@@ -94,6 +94,26 @@ ol[class] {
     margin-top: 0;
   }
 
+  & > h1 {
```

Comment:
What's the reason behind using child combinator `>` for h1, but **not using** it for `h2` and `h3`?

Reply:
MFU: remove child combinator